frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.styled.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from "@emotion/styled";
+import Button from "metabase/core/components/Button";
+import { color } from "metabase/lib/colors";
+
+const BACKGROUND_COLOR = "#12436e";
```

Comment:
I thought that custom colors should be automatically rejected by eslint. Can we use a similar existing color instead? Can it be computed based on some existing colors?